CMP1903_A1_2324/Program.cs

```diff
@@ -13,8 +13,32 @@ static void Main(string[] args)
             /*
              * Create a Game object and call its methods.
              * Create a Testing object to verify the output and operation of the other classes.
-             * Git Test
              */
+
+            bool testingMode = true;
+            Game activeGame = new Game();
+            Testing activeTester = new Testing();
+
+            if (testingMode)
+            {
+                activeTester.RunTest();
+            }
+            
+            // Blank line for output aesthetic.
+            Console.WriteLine();
+            
+            // Enable the starter game.
+            activeGame.StartRoll();
+            
+            // Check for next step.
+            Console.WriteLine("Would you like to roll more times? (Y/N): ");
+            string answer = Console.ReadLine().ToLower();
+
+            if (answer == "y")
+            {
+                activeGame.ExtraRolls();
+            }
+
```

Comment:
The program will auto-close after the extra rolls segment, consider adding another Console.ReadLine to stop this